### DISCUSSION TITLE
Sosyal Medyalarda Aktif kalanlar Ödüllendirilmesi...

### DISCUSSION THREAD

**Discussion by Skips1** [IBAX-io/go-ibax#280] [Ideas]
Hepimizin bildiği Discord ,Twiter,Telegram gibi Sosyal  alanlar Bu ve bunun gibi projelerde İnsan ietişimi olsun güven bilgi ve pek çoğu üzerinde Çok büyük önem arz ediyor.

Benim Önerim Bu platformlarda Aktif kalanlar ,insanlara yardım edenler için Level sistemi getirilmeli ve leveli çok olan kişileri ayrıcalık tanınmaları çok ama çok iyi olur Benim gibi Düşünen varsa Lütfen bana destek çıksın.  Ancak böyle Herkes hak ettiği kadar alır ve işimiz şansa kalmaz veya şu kırık yutupçuların gerisinde kalmayız...

**Comment by ghost** [comment]
Aynen öyle Daha önceki katıldığım projelerde de böyleydi herkes bir level kasma yarışındaydı çok iyiydi umarım olur
